jaxrs/resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/ResteasyClientBuilder.java

```diff
@@ -335,7 +336,7 @@ protected ResteasyProviderFactory getProviderFactory()
       if (providerFactory == null)
       {
          // create a new one
-         providerFactory = new ResteasyProviderFactory();

```

Comment:
This line can actually remain the same since ClientConfiguration constructor now instantiate a new LocalResteasyProviderFactory(). It will be consistent with ResteasyClientBuilder.withConfig() method where we also use ResteasyProviderFactory.  What do you think ?